ubiquity-fundraising-research-april-may-2024.md

```diff
@@ -0,0 +1,87 @@
+
+## Market and Competition
+In its materials, Ubiquity has not clearly shown which market it serves and what problems it solves. Listed problems and solutions do not distinguish Ubiquity from other products. The competitive environment for Ubiquity is incorrectly defined. For each of Ubiquity's product lines, it is necessary to define the competitive environment. Companies described as competitors based on the AI platform criterion are not direct competitors because Ubiquity is not an AI platform. An analysis found no AI products and services offered by Ubiquity. Using LLM via API does not place it alongside companies that train their own models or create solutions using complex prompts. Experts see a wave of projects trying to use the AI + crypto theme. If a project lacks AI competence like Ubiquity, such positioning is perceived negatively.
+
+Similar systems to those offered by Ubiquity are not popular on GitHub 
+
+[https://github.com/marketplace?type=apps&category=project-management](https://github.com/marketplace?type=apps&category=project-management) 
+
+However, similar systems exist, and some of them have more installations than Ubiquity:
+
+- Feature bot
+
+[https://github.com/marketplace/feature-bot](https://github.com/marketplace/feature-bot)
+
+[https://github.com/feature-sh](https://github.com/feature-sh)
+
+[https://feature.sh/](https://feature.sh/)
+
+[https://docs.feature.sh/](https://docs.feature.sh/)
+
+
+## Products and Users
+
+### Users
+Experts note the near absence of users and partners, which does not provide an opportunity to assess traction.
+
+### Product Portfolio
+As Ubiquity has practically no clients, there is no reason to believe that Ubiquity has found a product-market fit. Some investors speculated that Ubiquity does not have a clear vision and changes its course. Other experts noted discrepancies and incompleteness of information in materials, on the website, on GitHub, and on social networks of Ubiquity.
+
+### Stablecoin
+The presence of a stablecoin as a product requires quality documentation. As an example of documentation, one could use the documentation of Ethena [https://ethena-labs.gitbook.io/ethena-labs](https://ethena-labs.gitbook.io/ethena-labs). An unregulated MiCA stablecoin will reduce interest from investors who finance companies in the EEA territory.
+
+### Community and Marketing
+It was difficult to understand what exactly Ubiquity does, as information in the presentation, on the website, on GitHub, and on social networks differs. Twitter score has a rating of 5 (bad) [https://twitterscore.io/twitter/UbiquityDAO/overview/](https://twitterscore.io/twitter/UbiquityDAO/overview/). A strong community supporting the project was not found, nor were there marketing activities that create interest and hype for the project.
```

Comment:
I generally explain that its a neobank offering financial services, with a DAO operating system application sitting on top. 

The application is used as a tool to get other organizations to "bank" with us. 

Any idle capital in our system we make yield on.